Gifski/Gifski.swift

```diff
@@ -49,6 +49,17 @@ final class Gifski {
 	// This is carefully nil'd and should remain private.
 	private static var gifData: NSMutableData?
 
+	private static var writeCallback: ((Int) -> Void)?
+	private static var onNewFrame: ((Int64) -> Void)?
+
+	static func setWriteCallback(_ callback: ((Int) -> Void)?) {
+		writeCallback = callback
+	}
+
+	static func setOnNewFrame(_ callback: ((Int64) -> Void)?) {
+			onNewFrame = callback
+		}
```

Comment:
These methods are moot. It's clearer to just assign the closure directly as it only supports one closure.